docs/appendix/message-transport-mechanism.md

```diff
@@ -86,7 +90,22 @@ A rejection reason will be added to reject a message with an invalid version
 
 ### Service discovery
 
-In the service discovery stage, DNS[^16] is used to discover the capabilities as well as the endpoint host name and IP address of the remote participant.
+The purpose of the discovery stage is to discover the endpoint, public-key for signing and, optionally, the capabilities (e.g. supported UFTP versions) of the other participants.  
```

Comment:
I think it's good to use the new name (ShapeShifter) instead of UFTP

Reply:
updated